application-forum-ui/src/main/resources/ForumCode/Macros.xml

```diff
@@ -479,10 +479,24 @@
   #end
 #end
 ##
+#macro(checkIfHostNameIsValid $stringURL $result)
+  ## @since 12.8RC1
+  #set ($hostName = $urltool.getURLFromString($stringURL).getHost())
```

Comment:
Maybe a more explicit message? e.g. `#getURLFromString is available from 12.8RC1`?